components/input/index.en-US.md

```diff
@@ -26,7 +26,7 @@ Keyboard and mouse can be used for providing or changing data.
 | prefix | The prefix icon for the Input. | string\|ReactNode |  |
 | size | The size of the input box. Note: in the context of a form, the `large` size is used. Available: `large` `default` `small` | string | `default` |
 | suffix | The suffix icon for the Input. | string\|ReactNode |  |
-| type | The type of input, `text` or `textarea`(`type=textarea` are deprecated after `2.12`, please use `Input.TextArea`) | string | `text` |
+| type | The type of input (`type=textarea` has removed after `3.0`, please use `Input.TextArea`) | string | `text` |
```

Comment:
Would this be better kept just in the changelog? I think this will be confusing to have in the documentation for input